artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/tasks/components/task-details/task-details.component.ts

```diff
@@ -95,26 +95,36 @@ export class TaskDetailsComponent implements OnInit {
       .pipe(
         finalize(() => {
           this.#isLoading.set(false);
-          this.isSuggestionsVisible.set(true);
-          setTimeout(() => this.scrollToElement(this.taskEditor), 50);
+          this.editorStateServiceService.suggestionsVisible = true;
+          setTimeout(() => this.scrollToElement(this.taskEditor), 10);
```

Comment:
sprawdz czemu nie dziala bez setTimeouta, jezeli to przez to ze np nie ma elementu this.taskEditor to trzeba sie zapiac na jego zaladowanie a nie tu